content/300-guides/050-database/100-developing-with-prisma-migrate/100-advanced-migrate-scenarios.mdx

```diff
@@ -96,15 +96,15 @@ You can use the same technique to rename a `model` - edit the generated SQL to _
 
 ## Using the expand and contract pattern to evolve the schema without downtime
 
-One of the challenges when renaming fields is that it can lead to downtime in the time frame between applying the rename migration and deloying a new version of the application code which uses the renamed field.
+One of the challenges when making schema changes to existing fields, e.g., renaming a field, is that it can lead to downtime in the time frame between applying the migration that modifies and deloying a new version of the application code which uses the modified field.
```

Comment:
Super picky but i ran out of breath reading that 😅. I think an extra comma after `migration that modifies` would do the trick. So

> One of the challenges when making schema changes to existing fields, e.g., renaming a field, is that it can lead to downtime in the time frame between applying the migration that modifies, and deloying a new version of the application code which uses the modified field.

Reply:
Good point! That was a mouthful. 

I rephrased it to make it a bit more readable:
> Making schema changes to existing fields, e.g., renaming a field can lead to downtime. It happens in the time frame between applying a migration that modifies an existing field, and deploying a new version of the application code which uses the modified field.